### DISCUSSION TITLE
Erro with DB configuration

### DISCUSSION THREAD

**Discussion by jbassoz** [Nereziel/cs2-WeaponPaints#105] [Q&A] [answered]
I need help, this code is appearing on my website. I uploaded the site to a localhost to use on a LAN server with friends, but the error persists. I don't know what to do.

`Fatal error: Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'skins.wp_player_skins' doesn't exist in C:\xampp\htdocs\skins\class\database.php:18 Stack trace: #0 C:\xampp\htdocs\skins\class\database.php(18): PDOStatement->execute(Array) #1 C:\xampp\htdocs\skins\index.php(14): DataBase->select('SELECT `weapon_...', Array) #2 {main} thrown in C:\xampp\htdocs\skins\class\database.php on line 18
`

**Comment by mrc4tt** [comment]
It sounds like it has not created data from tables in the database.

As it says: The table does not exist. ^

**Comment by L1teD** [comment, marked as answer]
Connect plugin to db, plugin will automatically create tables
Then you can use website normally
